covid19-dashboard/main.py

```diff
@@ -33,33 +35,45 @@
 
 app.config.from_mapping(config)
 
-# Load the configuration file from the instance folder.
-app.config.from_pyfile("config.py")
 cache = Cache(app)
 
-# Retrieves DataCommons Server from configuration file.
-DC_SERVER = app.config["DC_SERVER"]
-
-# Each place_type has its own stat_var.
-# Get the dictionary from the configuration file.
-STAT_VARS = app.config["STAT_VARS"]
+# DataCommons Host Server.
+DC_SERVER = "https://api.datacommons.org/"
 
 # COMMONLY-USED TYPES.
 KeyToTimeSeries = Union[Dict[str, int], int, str]
 GeoIdToDataType = Dict[str, KeyToTimeSeries]
 GeoIdToStatsType = Dict[str, Dict[str, int]]
 PlaceToInfoType = Dict[str, Dict[str, str]]
 
-@app.route("/api/data/<string:geo_id>")
+
+@app.route("/api/data/<string:geo_id>/<string:dashboard_id>")
 @cache.cached(timeout=3600)
-def county_data(geo_id: str):
+def data(geo_id: str, dashboard_id: str):
     """
-    Returns any placeType's datta.
+    Returns any placeType's data.
     NOTE: for return type documentation, please see README.md's APIs section.
     :return: geo_id->{**key_to_timeseries}.
     """
+    dashboard_id = dashboard_id or "covid19"
+    print(dashboard_id)
+
+    # How many days should we perform the moving average for?
+    # 0 == None
+    moving_averages_chunks = {"covid19": 7, "socialWellness": 0}
```

Comment:
maybe add a todo for these options to be config driven

Reply:
Done.